__tests__/app.test.js

```diff
@@ -115,4 +115,20 @@ describe("Sepcific comments request by review ID /api/reviews/:review_id/comment
         });
       });
   });
+  test("returns 200 status with message of no comments when valid review_id is provided but there are no comments", () => {
+    return request(app)
+      .get("/api/reviews/6/comments")
+      .expect(200)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("No Comments found for this review id");
+      });
+  });
+  test("status 404 when bad request has been made", () => {
+    return request(app)
+      .get("/api/reviews/40/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("No such path found. Try again...");
+      });
+  });
```

Comment:
Missing 400 test, what happens if i make a request to /api/reviews/notAnId/comments?